shared/editor/embeds/index.tsx

```diff
@@ -621,6 +624,17 @@ const embeds: EmbedDescriptor[] = [
     icon: <Img src="/images/youtube.png" alt="YouTube" />,
     component: YouTube,
   }),
+  /* The generic iframe embed should always be the last one */
+  new EmbedDescriptor({
+    title: "Iframe",
```

Comment:
```suggestion
    title: "Embed",
```

This seems like better language for the average user, iframe can remain as a search keyword.